perms/manager.go

```diff
@@ -96,6 +101,18 @@ func NewManager(withAllSubServers bool) (*Manager, error) {
 	}, nil
 }
 
+// IsWhiteListedURL returns true if the given URL has been whitelisted meaning
+// that it does not require a macaroon for validation. A URL is considered
+// white-listed if it has no operations associated with a URL.
+func (pm *Manager) IsWhiteListedURL(url string) bool {
+	pm.permsMu.Lock()
+	defer pm.permsMu.Unlock()
+
+	ops, ok := pm.perms[url]
+
+	return ok && len(ops) == 0
```

Comment:
is this an ok axiom?

Reply:
compared to a previous version of this pr which kept an explicit whitelist. Just realised it might not be needed if this assumption is ok